### PR TITLE
Add mixer override passthrough option

### PR DESCRIPTION
**Pull request by gongtao0607**
This option allows user to easily test mixer geometry settings by using stick input.

Test performed on a Matek G474-Heli (with Lua sending this MSP)

### CODE REVIEW

**Comment by gongtao0607** [comment]
This PR is to help simplify this step in mixer setup:

>You might want to temporally increase the I gains (to around 200) on cyclic and move the cyclic stick to find the limits. MAKE SURE TO TURN THEM BACK
https://www.rotorflight.org/docs/Wiki/Configurator/Mixer#cyclic-blade-pitch-limit

File: src/main/flight/mixer.c [hunk 1]
```diff
@@ -192,13 +192,50 @@ static inline void mixerApplyInputLimit(int index, float value)
...
+        break;
+    default:
+        return value;
+    }
+
+    // Scale rc by 120% for easier observing endpoints.
+    rc *= 1.2f;
+
+    if (rc > 0) {
+        value = scaleRangef(rc, 0, 1.0f, 0, mixerInputs(index)->max / 1000.0f);
```
Comment (rotorflight) [inline src/main/flight/mixer.c:221]: Please don't reuse function arguments as variables. IT's a bad habit.

**Review by rotorflight — CHANGES_REQUESTED** [review]
Fix variable naming.

**Comment by rotorflight** [comment]
How is this supposed to be used?

File: src/main/flight/mixer.c (hunk 1 above)
Comment (gongtao0607) [reply src/main/flight/mixer.c]: Resolved

**Comment by gongtao0607** [comment]
> How is this supposed to be used? Is the Configurator support coming?

It should be a toggle box in Lua and configurator mixer page. I have an updated Lua to support this. I don't know how to modify configurator so far.

**Comment by robthomson** [comment]
Which lua?

Not seen a or for that yet.

**Comment by gongtao0607** [comment]
> Which lua?
> 
> Not seen a or for that yet.

In my own branch:
https://github.com/rotorflight/rotorflight-lua-scripts/compare/master...gongtao0607:rotorflight-lua-scripts:mixing

**Comment by robthomson** [comment]
Cool.  That explains it then :)

**Review by rotorflight — APPROVED** [review]

**Pull request merged**